boards/nucleo-f767zi/include/periph_conf.h

```diff
@@ -208,6 +208,7 @@ static const adc_conf_t adc_config[] = {
 };
 
 #define VBAT_ADC            ADC_LINE(6) /**< VBAT ADC line */
+#define ADC_CLK_MAX         MHZ(36)
```

Comment:
```suggestion
#define VBAT_ADC            ADC_LINE(6) /**< VBAT ADC line */
#define ADC_CLK_MAX         MHZ(36)     /**< Use a faster than default ADC clock */
```

The CI wants the macro commented. I think in this instance it is actually helpful to indicate that the ADC clock here is running at a higher than usual clock

Reply:
I agree. I accepted the change via a fixup commit.